doc/ADMIN.md

```diff
@@ -1,10 +1,18 @@
-* The app relies on YunoHost's LDAP server for users to log in:
+# Connection
+
+The app relies on YunoHost's LDAP server for users to log in:
   * Standard users need the `main` permission;
   * Users with the `admin` permission will have access to the administration panel.
 
-* The app can access YunoHost's multimedia directories:
-  
-  Choose one of the folders in `/home/yunohost.multimedia/share` upon configuration of your libraries.
+# Multimedia folders
+
+The app can access YunoHost's multimedia directories:
+
+Choose one of the subfolders in `/home/yunohost.multimedia/share` upon configuration of your libraries.
```

Comment:
```suggestion
If you want to use them, choose one of the subfolders in `/home/yunohost.multimedia/share` upon configuration of your libraries.
```
Great idea for this pr! I’d only make this sentence conditional because it’s not obligatory to use them (to be changed on the other placed too, if that make sence) :)